src/kafkajsWinstonLogger.ts

```diff
@@ -0,0 +1,33 @@
+import { logCreator, LogEntry, logLevel } from 'kafkajs'
+import winston from 'winston'
+import { transports } from './logger'
+
+const mapKafkajsLogLevelToWinstonLogLevel = (level: logLevel): string => {
+    switch (level) {
+        case logLevel.ERROR:
+        case logLevel.NOTHING:
```

Comment:
Er NOTHING en feil? Bare lurer om det kan bli mange feil, om denne brukes mye

Reply:
Du vet, Nothing is wrong! Men veldig godt spørsmål - jeg har kopiert dette fra kafkajs-siden: https://kafka.js.org/docs/custom-logger. Har ikke sett noen meldinger i loggen da så inntil det motsatte er bevist så lar jeg det stå.